docker-compose.yml

```diff
@@ -23,9 +26,8 @@ services:
   db:
     image: postgres:latest
     env_file: .env
-    ports:
-      - "5432:5432"
   redis:
     image: redis:alpine
-    ports:
-      - "6379:6379"
+    command: redis-server --save ""  
+    volumes:
+      - "./redis-data:/data"
```

Comment:
abrichr 9 days ago Member
What is the reason for this change?

Reply:
https://stackoverflow.com/questions/58903982/redis-is-configured-to-save-rdb-snapshots-but-it-is-currently-not-able-to-persi

Reply:
It was working before this change. What is the reason this line is now needed?

Reply:
Update: https://github.com/MLDSAI/VisionTracing/pull/2/commits/0b21322d56bb15eb5a511762a4264aa686f001c7

Since I disabled snapshotting (as you can see in this discussion: https://github.com/MLDSAI/VisionTracing/pull/2#discussion_r488857892), this is no longer necessary, and I removed these lines from the docker-compose file.